### ISSUE TITLE
No slides showing

### ISSUE THREAD

**Issue by charleskorn** [binford2k/showoff#706]
I ran:

```
gem install showoff
showoff init hello-world
cd hello-world/
showoff serve
```

Opening `http://localhost:9090/presenter` gives me an empty display: http://pasteboard.co/42k8jkus6.png

This happens in both Chrome 56 and Firefox 52. 

In Chrome, I get these debugging messages in the console:

```
presenter.js?v=0.17.0:928 Setting layout to beside
jquery.cycle.all-2.8.0.js:28 [cycle] terminating; too few slides: 0
showoff.js:910 Control socket opened
showoff.js:1096 New slide: null
showoff.js:990 Not a presenter! TypeError: Cannot read property 'scrollIntoView' of undefined
```

It looks like the `/slides` endpoint is returning an empty response. There are no warning or error messages in the terminal where I'm running `showoff serve`.

Switching to Showoff version 0.16.2 solves the problem. 

What am I doing wrong?

**Comment by charleskorn** [comment]
Some version information, if it's useful:

```
$ ruby --version
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin16]
$ gem --version
2.6.8
$ showoff --version
showoff version 0.17.0
```

**Comment by binford2k** [comment]
Thanks @charleskorn. This was fixed in #693 and I'll be pushing a new release today or tomorrow.

In the meantime, change your `showoff.json` from using `section` keys to just listing the directories in an array:

```json
{
  "name": "My Preso",
  "sections": [
    "one"
  ]
}
```

**Comment by charleskorn** [comment]
Thanks for the quick response @binford2k, it's working properly with that change.
